packages/ckeditor5-list/src/liststyleediting.js

```diff
@@ -204,11 +235,117 @@ export default class ListStyleEditing extends Plugin {
 	}
 }
 
-// Returns a converter that consumes the `style` attribute and searches for the `list-style-type` definition.
+// Strategy for dealing with listItem attribute supported by this plugin.
+//
+// @typedef {Object} module:list/liststyleediting~AttributeStrategy
```

Comment:
If you want to use this type in public docs, you got to make it public. ATM JSDoc fails

```
JSDoc started...
Incorrect param type: Array.<module:list/liststyleediting~AttributeStrategy>
        at /Users/oleq/ck/5/ckeditor5/packages/ckeditor5-list/src/liststyleediting.js:137
```

Maybe it could be @Protected since it is used internally and does not make sense in the public API? Not sure if this is possible in JSDocs.

Reply:
It's private here. But JSDoc doesn't see line comments, only `/** ... */`.